docs/guide/users.md

```diff
@@ -0,0 +1,54 @@
+# Users and user roles
+
+A **"user"** in Apostrophe is an account (usually for a human or benevolent robot) that can be used to log into the website. Beyond that, users get specific sets of permissions based on the **role** they are assigned. More on that to come.
```

Comment:
```suggestion
A **"user"** in Apostrophe is an account (usually for a human or benevolent robot) that can be used to log into the admin interface. Beyond that, users get specific sets of permissions based on the **role** they are assigned. More on that to come.
```

Reply:
or similar. I want to make sure its distinct from 'site user' (like commerce for example)